app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, meaning it is shared across all instances of `Animal` and its subclasses. If this is not the intended behavior, consider using an instance variable instead.